backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/UserModel.java

```diff
@@ -21,6 +21,8 @@ public class UserModel extends BaseResourceModel<UserModel> {
 
 	private List<UserAttributeModel> userAttributes;
 
+	private List<SubscriptionModel> subscriptions;
```

Comment:
This should be removed. Subscriptions should be sent back to the consumer via HATEOAS links, instead.